data/installed_data.go

```diff
@@ -25,19 +33,62 @@ func NewInstalledDeisData(ri *k8s.ResourceInterfaceNamespaced) InstalledData {
 
 // Get method for InstalledDeisData
 func (g *installedDeisData) Get() ([]byte, error) {
-	rcItems, err := k8s.GetReplicationControllers(g.k8sResources.ReplicationControllers())
+	var cluster models.Cluster
+	deployments, err := k8s.GetDeployments(g.k8sResources.Deployments())
 	if err != nil {
 		return nil, err
 	}
-	var cluster models.Cluster
-	for _, rc := range rcItems {
-		component := models.ComponentVersion{}
-		component.Component = &models.Component{}
-		component.Version = &models.Version{}
-		component.Component.Name = rc.Name
-		desc := rc.Annotations["chart.helm.sh/description"]
-		component.Component.Description = &desc
-		component.Version.Version = rc.Annotations["chart.helm.sh/version"]
+	for _, deployment := range deployments {
+		component := models.ComponentVersion{
+			Component: &models.Component{
+				Name: deployment.Name,
+				Type: &deploymentType,
+			},
+			Version: &models.Version{
+				Version: deployment.Annotations[versionAnnotation],
+				Data: &models.VersionData{
+					Image: &deployment.Spec.Template.Spec.Containers[0].Image,
+				},
+			},
+		}
+		cluster.Components = append(cluster.Components, &component)
+	}
+	daemonSets, err := k8s.GetDaemonSets(g.k8sResources.DaemonSets())
+	if err != nil {
+		return nil, err
+	}
+	for _, daemonSet := range daemonSets {
+		component := models.ComponentVersion{
+			Component: &models.Component{
+				Name: daemonSet.Name,
+				Type: &daemonSetType,
+			},
+			Version: &models.Version{
+				Version: daemonSet.Annotations[versionAnnotation],
```

Comment:
seems like the version annotation will always exist, but this map lookup should still be checked. seems like the correct thing is to log a warning if the key is missing, but proceed anyway